### PR TITLE
Update the outdated GDPR for analytics related documentation.

### PR DESCRIPTION
**Pull request by RAVEENSR**
## Purpose
Update the outdated GDPR for analytics related documentation.

general-data-protection-regulation-gdpr-for-wso2-api-manager-analytics

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/administer/product-security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md [hunk 1]
```diff
@@ -141,67 +141,6 @@ This tool is packaged with WSO2 API Manager by default. Follow the steps below t
 This tool can run standalone and therefore cater to multiple products. This means that if you are using multiple WSO2 products and need to delete the user's identity from all products at once, you can do so by running the tool in standalone mode.
 For information on how to build and run the Forget-Me tool, see [Removing References to Deleted User Identities in WSO2 Products](https://docs.wso2.com/display/ADMIN44x/Removing+References+to+Deleted+User+Identities+in+WSO2+Products) in the WSO2 Administration Guide.
```
Comment (Mariangela) [inline en/docs/administer/product-security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md:142]: The doc links should not point to the admin44x space anymore.

**Comment by Mariangela** [comment]
@RAVEENSR , after you have updated this based on the review comment pls send a PR to master as well, so that master is not out of sync.

File: en/docs/administer/product-security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md (hunk 1 above)
Comment (RAVEENSR) [reply en/docs/administer/product-security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md:182]: This section is not relevant to my PR. This section(the commented one) was not changed by PR. I have fixed my PR with the sections that are related to my change with 6c779f9. You may change the highlighted section.

**Comment by RAVEENSR** [comment]
> @RAVEENSR , after you have updated this based on the review comment pls send a PR to master as well, so that master is not out of sync.

Added to the master branch via https://github.com/wso2/docs-apim/pull/1113

**Pull request merged**